build.gradle

```diff
@@ -64,7 +72,7 @@ dependencies {
         exclude group: 'commons-pool2', module: 'commons-pool2'
         exclude group: 'xml-apis', module: 'xml-apis'
     }
-    compile("kg.apc:jmeter-plugins-webdriver:${jmPluginVersion}"){
+    compile("kg.apc:jmeter-plugins-webdriver:2.3"){
```

Comment:
@double16 why are you going to a fixed webdriver version here?

Reply:
Maybe this line needs an update instead?
https://github.com/jmeter-gradle-plugin/jmeter-gradle-plugin/blob/787fc6a9a60fbc99d7daf3a05cb867ce0da0fef2/build.gradle#L19

Reply:
This particular plugin has departed from the common version number of the other plugins. The others have a latest version of '1.4.0'. The `jmeter-plugins-webdriver` is actually `3.0` now. Do you think it's better to stay at `1.4.0` for this one too and let the user of the plugin change the version number?

Reply:
In which case, what you did makes sense 👍